src/Modules/Reports/Commands.Reports/help/MicrosoftPowerBIMgmt.Reports.md

```diff
@@ -12,10 +12,10 @@ PowerShell cmdlets for managing and interacting with Power BI reports.
 
 ## MicrosoftPowerBIMgmt.Reports Cmdlets
 ### [Copy-PowerBIReport](Copy-PowerBIReport.md)
-Creates a copy of the specified report.
```

Comment:
Do you want this in the PR?

Reply:
Yeah just because master was out of date with code